webapp/tests/Unit/Utils/UtilsTest.php

```diff
@@ -713,6 +713,54 @@ public function provideTestGetImageSize(): Generator
         yield [__DIR__ . '/../../../public/images/DOMjudgelogo.svg', 510, 1122];
     }
 
+    public function testSanitizeSvg(): void
+    {
+        $dirty = <<<EOF
```

Comment:
Can you add the URL to the original location so we can update if there is more nasty stuff?

Reply:
Isn't the URL a line above your comment? :-D